specs/capella/light-client/full-node.md

```diff
@@ -76,3 +76,31 @@ def block_to_light_client_header(block: SignedBeaconBlock) -> LightClientHeader:
         execution_branch=execution_branch,
     )
 ```
+
+## Deriving light client data
+
+### `create_light_client_update`
+
+Full nodes SHOULD continue deriving the `LightClientUpdate` for each sync committee period as per the altair specs but with the modified or upgraded `attested_header` and `finalized_header` with the following considerations:
+
+- Both `attested_header` and `finalized_header` are of the same fork types as `LightClientUpdate` and might need upgradation to bundle as update.
+- Above implies that the first update of capella MIGHT have both `attested_header` and `finalized_header` upgraded (if capella hardfork coincides with a new period) as they might belong to pre-capella epochs. Post this `finalized_header` will need upgradation till a new finalization occurs in capella.
+- Over the wire, the fork information of transmitted data might be available (like fork `version` in beacon-apis or fork `context` in req/resp or fork gossip `topics`)
```

Comment:
It is always available, otherwise the protocol doesn't work.

libp2p req/resp uses the `ForkDigest`-context: https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/p2p-interface.md#forkdigest-context

libp2p gossip has the `ForkDigestValue` in the topic name: https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#topics-and-messages

REST uses a mix of Eth-Consensus-Version HTTP header, binary interleaved ForkDigest, or JSON "version" fields to indicate spec fork.